_pages/ato/checklist.md

```diff
@@ -35,9 +35,9 @@ Many of these tasks can be done in parallel...this is just a suggestion of prior
 1. [ ] [Security scans](https://pages.18f.gov/before-you-ship/security/scanning/)
     * [ ] Set up [static analysis](https://pages.18f.gov/before-you-ship/security/static-analysis/) service
         * [ ] Add service badges to the README
-    * [ ] [Run OWASP ZAP](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/)
+    * [ ] [Perform automated vulnerability testing](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/)
```

Comment:
Is automated vulnerability testing required for ATO right now? Or is manual vulnerability testing acceptable right now? If manual is acceptable, this should probably be "Perform vulnerability testing" (and let that more detailed page explain the requirements).